frontend/lit/components/Reference.js

```diff
@@ -12,8 +12,7 @@ const markKeywords = (text, allSettings) => {
         const all_tokens = allSettings.set1.keywords
                 .concat(allSettings.set2.keywords)
                 .concat(allSettings.set3.keywords),
-            all_re = new RegExp(all_tokens.join("|"), "gim");
-
+            all_re = new RegExp(all_tokens.join("|").replace(/\*/g, ".*?"), "gim");
```

Comment:
I'm not sold on this idea, but what do you think about wrapping word boundaries? It's nice because it'll highlight the whole word, not just the phrase, and we may get fewer false-positives.

<img width="369" alt="image" src="https://github.com/shapiromatron/hawc/assets/999952/91eb649d-cde0-4407-9f2f-c93bf8ffb3b5">

Without the word boundary, we're basically getting star-style matches all the time: 

<img width="370" alt="image" src="https://github.com/shapiromatron/hawc/assets/999952/36e7caab-5320-4f26-a009-496613fabb18">

Looping in @swatford  as well for thoughts...